cs/src/Contracts/TunnelListResponse.cs

```diff
@@ -19,14 +19,14 @@ public class TunnelListResponse
     /// </summary>
     public TunnelListResponse()
```

Comment:
I think `TunneListResponse` is not used anywhere now. We should use `TunnelListByRegionResponse` for the response from the new Azure version of the list tunnels API.

Reply:
removed